helm/hpcc/templates/daemonset.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.global.prepull -}}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: prepull
+spec:
+  selector:
+    matchLabels:
+      name: prepull 
+  template:
+    metadata:
+      labels:
+        name: prepull 
+    spec:
+      containers:
+      - name: prepull 
+{{ include "hpcc.addImageAttrs" (dict "root" $ "me" . "imagename" "platform-core") | indent 8 }}
+        command: ["sleep", "10000d"]
```

Comment:
you'll have retired before it times out?
Will this continue to poll to see if there are new images?

How does this approach compare with https://gist.github.com/itaysk/7bc3e56d69c4d72a549286d98fd557dd